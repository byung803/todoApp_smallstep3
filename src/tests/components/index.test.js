import React from 'react';
import { shallow } from 'enzyme';
import Index from '../../components';

test('should test components index', () => {
    const wrapper = shallow(<Index />);
    expect(wrapper).toMatchSnapshot();
});