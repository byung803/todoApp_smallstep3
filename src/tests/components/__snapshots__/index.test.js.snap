// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test components index 1`] = `
<div>
  template 오늘은 뭘 만들까? 두근 두근
</div>
`;
